server/leader.go

```diff
@@ -292,6 +292,21 @@ func (s *Server) campaignLeader() error {
 			log.Info("server is closed")
 			return nil
 		}
+
+		// leader apply config
+		configHash, err := s.kv.LoadConfigHash()
```

Comment:
Leader no need to reload, it [will be set](https://github.com/pingcap/pd/blob/master/server/server.go#L567-L575) firstly then persist to KV.